packages/contracts-bedrock/src/L1/OPStackManager.sol

```diff
@@ -55,6 +55,9 @@ contract OPStackManager is ISemver, Initializable {
         uint32 basefeeScalar;
         uint32 blobBasefeeScalar;
         uint256 l2ChainId;
+        // The correct type is AnchorStateRegistry.StartingAnchorRoot[] memory,
+        // but OP Deployer does not yet support structs.
+        bytes startingAnchorRoots;
```

Comment:
Do you think keeping this as `bytes` is ok, or should this be a TODO?

Reply:
Unless it becomes a headache, I think it's OK to live with this. So far it doesn't seem to affect the external interface too much.